checkout.md

```diff
@@ -0,0 +1,250 @@
+> rather than lesson discussion sessions.
+> We hope that this will give trainees a chance to meet more members of our community,
+> and get answers to questions about a broader range of topics.
+> Your trainer will let you know if this option is being offered to your group;
+> if it is,
+> please see our [Community Calendar]({{page.calendar}})
+> for the times of upcoming debriefing sessions.
+> You are still required to familiarize yourself with one particular lesson before taking part,
+> and to bring questions about it to discuss.
+
+### Rubric
+<a name="rubric"></a>
+
+Trainees should be prepared to answer these questions during the discussion session:
+
+*   Have you been a learner, helper, or instructor at a workshop before?
+    If so, which one(s), and what were the most interesting or surprising things you learned?
+*   What pre- and during-workshop responsibilities does each instructor have?
+*   What lesson have you selected for your demonstration lesson?
+*   Have you tested installation of the required software?
+*   How long do you think it will actually take to teach?
+    (Alternatively, how far do you actually expect to get?)
+*   Are you adapting or modifying the lesson material?
+    If so, how and why?
+*   Are there things in the material that you don't understand or feel uncertain about?
+*   Where in the material learners are most likely to misunderstand things or get lost,
+    and why?
+*   What are you most excited about teaching? (Be specific.)
+*   What are you most apprehensive about teaching? (Be specific.)
+
+(Note that any particular trainee will only be asked some of these questions,
+but they should prepare answers to all of them.)
+
+## Part 3: Teach a Short Demonstration Lesson
+<a name="demo-lesson"></a>
+
+Once you have completed your discussion session,
+the final step is to sign up for a 5-minute online demonstration lesson
+[this Etherpad]({{page.demopad}})
+(or add yourself to the top of the pad if none of the available times work for you).
+Like the discussion sessions,
+these demonstration lessons are run as video conferences using Google Hangouts,
+and the instructor leading the session will post connection details and other information
+in [the Etherpad]({{page.demopad}})
+shortly before it begins.
+
+For your teaching demonstration,
+will ask you to teach a short segment from your chosen lesson.
+If you want to do your final demonstration(s) in a language other than English,
+please let us know in advance so that we can find someone to moderate.
+
+> Note that we will pick the segment of the lesson you are to teach
+> on the day of the demonstration,
+> so you must be prepared to teach any part of your chosen lesson.
+
+For your demonstration,
+you will screen-share through Google Hangouts
+and live code as if your computer was plugged into a projector
+and you were standing in front of a class.
+You may use whatever notes you want,
+provided they are not visible on your screen.
```

Comment:
I think that once someone has enlarged their font enough for the screen 
to be readable when projected, side-by-side is impossible (at least on a 
single-screen system - if they're lucky enough to have two projectors, 
that changes everything).